styles.css

```diff
@@ -0,0 +1,70 @@
+
+html {
+    background-image: url('sunray.jpeg');
+    background-size: cover;
+}
```

Comment:
try and make your css blocks look like this one! where there's a space after the selector (`html`) and before the open bracket (`{`), and there are spaces between the semi colons and the values. you can find a linter on the vs code store that could auto format css and use that!